core/src/main/java/hudson/model/Queue.java

```diff
@@ -1419,7 +1423,13 @@ public void maintain() {
                     Collections.sort(blockedItems, QueueSorter.DEFAULT_BLOCKED_ITEM_COMPARATOR);
                 }
                 for (BlockedItem p : blockedItems) {
+                    String taskDisplayName = p.task.getFullDisplayName();
+                    LOGGER.log(Level.FINER, "Current blocked item: {0}", taskDisplayName);
                     if (!isBuildBlocked(p) && allowNewBuildableTask(p.task)) {
+                        LOGGER.log(Level.INFO,
```

Comment:
Why is this at info - esp when entering the blocked state is `FINEST`? 
The log levels should be symetrical.